resources/js/processes-catalogue/components/RequestTab.vue

```diff
@@ -168,8 +167,10 @@ export default {
     },
     queryBuilder() {
       let pmql = "";
-      if (this.pmqlRequest !== undefined) {
-        pmql = this.pmqlRequest;
+      if (this.pmqlRequest !== undefined && this.pmqlRequest !== "") {
+        pmql = this.pmqlRequest + " AND process_id=" + `${this.process.id}`;
+      } else {
+        pmql = " process_id=" + `${this.process.id}`;
```

Comment:
`pmqlRequest` does not exist, there will be an error

Reply:
Solved. Only pmql variable is used. The If ELSE code was removed

Reply:
solved